src/lib/features/scheduler/scheduler-service.ts

```diff
@@ -22,7 +29,9 @@ export class SchedulerService {
         this.intervalIds.push(
             setInterval(async () => {
                 try {
-                    if (this.mode === 'active') {
+                    const maintenanceMode =
+                        await this.maintenanceService.isMaintenanceMode();
+                    if (this.mode === 'active' && !maintenanceMode) {
```

Comment:
I think we decided to remove these states

Reply:
Addressed in https://github.com/Unleash/unleash/pull/5363/commits/7c52e3f63826fe9a7c572b959252fc042a0a0cf2